### PR TITLE
Set up ID analyzer for id and isil fields

### LINKED ISSUE #131: Enable truncation of ISILs
Currently, you can't use truncation searching the `isil` field. This would be useful to filter by sub-libraries of a specific library e.g. for the many Institutsbibliotheken of the ULB Münster http://beta.lobid.org/organisations/search?q=isil:DE-6*.

### PR DESCRIPTION
**Pull request by fsteeg**
Will resolve #131

Uses keyword tokenizer and lowercase filter to work with query string query's lowercasing. Update tests for tweaked behavior.

### CODE REVIEW

File: conf/index-settings.json [hunk 1]
```diff
@@ -43,7 +47,11 @@
...
 				}, 
 				"id": {
-					"index": "not_analyzed", 
+					"analyzer": "id",
+					"type": "string"
+				},
+				"isil": {
+					"analyzer": "id",
 					"type": "string"
 				}
```
Comment (dr0i) [inline conf/index-settings.json]: Why is 'isil' not also set to 'not_analyzed', as with 'id' ?

Comment (fsteeg) [inline conf/index-settings.json]: They are both set to `analyzer: id`, the `not_analyzed` for the id field was removed.

The reason is to have the same behavior for both (to work both with upper- and lowercase).

Comment (dr0i) [inline conf/index-settings.json]: Ah, yes, I missed that the 'not_analyzed' was in fact removed .

**Comment by dr0i** [comment]
+1

**Pull request merged**